istio-proxy/update.sh

```diff
@@ -13,8 +17,7 @@ while getopts ":p:" opt; do
     esac
 done
 
-[[ -z "${PROXY_SHA}" ]] && PROXY_SHA="$(grep '%global proxy_git_commit ' istio-proxy.spec | cut -d' ' -f3)"
-
+PROXY_SHA=${PROXY_SHA-"$(grep '%global proxy_git_commit ' istio-proxy.spec | cut -d' ' -f3)"}
```

Comment:
nit: why not also use `awk` here?

Reply:
that's not the focus of this pr. I only changed strictly what was necessary for update.sh to succeed with my changes. Further improvements can be done in another PR.

Reply:
but you did change it in the top of tile from `cut` to `awk`?

Reply:
I agree with Daniel. Cut is fragile. Not sure it's strictly necessary for the first case, but regardless it cleans everything up. Lets do all or nothing.